go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/envoyproxy/xds-relay
+
+go 1.12
+
+require (
+	github.com/envoyproxy/go-control-plane v0.9.4
+	github.com/onsi/ginkgo v1.12.0
+	github.com/onsi/gomega v1.9.0
+	go.uber.org/zap v1.14.0
+	google.golang.org/grpc v1.27.1
```

Comment:
Do we need to version the grpc version? does go-control-plane do that for us?

Reply:
We need to instantiate our own gRPC server (which gets passed to go-control-plane), so we do need to track our own versioning:
https://github.com/envoyproxy/xds-relay/pull/11/files#diff-6bf74a2dac7bcac1a44aa2e114bada9cR37